ocfweb/error404/templates/404.html

```diff
@@ -0,0 +1,124 @@
+<!doctype html>
+<!--
+      Think this site is hideous? We could use your help!
+      More info at https://www.ocf.berkeley.edu/about/staff :-)
+
+      This is ocfweb version 2018-03-06-T21-08-09-git4c65c52
+      https://github.com/ocf/ocfweb/
+-->
```

Comment:
sort of confused why this duplicates our base template; is it possible to build this so it extends from the base? (see how other pages do it for an example)